packages/pluggableWidgets/maps-web/src/components/__tests__/GoogleMap.spec.ts

```diff
@@ -31,10 +45,15 @@ describe("Google maps", () => {
         initialize();
     });
 
```

Comment:
Can we move `mockUseLoadScriptHookWithReturn({ isLoaded: true, loadError: undefined });` to a `beforeEach` method? Then we avoid repetition.

Reply:
fixed